variable_set_test.go

```diff
@@ -286,80 +236,33 @@ func TestVariableSetsRead(t *testing.T) {
 func TestVariableSetsUpdate(t *testing.T) {
 	client := testClient(t)
 	ctx := context.Background()
-
-	orgTest, orgTestCleanup := createOrganization(t, client)
-	t.Cleanup(orgTestCleanup)
-
-	vsTest, _ := createVariableSet(t, client, orgTest, VariableSetCreateOptions{
-		Name:        String("OriginalName"),
-		Description: String("Original Description"),
-		Global:      Bool(false),
-	})
-
-	t.Run("when updating a subset of values", func(t *testing.T) {
-		options := VariableSetUpdateOptions{
-			Name:        String("UpdatedName"),
-			Description: String("Updated Description"),
-			Global:      Bool(true),
-		}
-
-		vsAfter, err := client.VariableSets.Update(ctx, vsTest.ID, &options)
-		require.NoError(t, err)
-
-		assert.Equal(t, *options.Name, vsAfter.Name)
-		assert.Equal(t, *options.Description, vsAfter.Description)
-		assert.Equal(t, *options.Global, vsAfter.Global)
-	})
-
-	t.Run("when options has an invalid variable set ID", func(t *testing.T) {
-		vsAfter, err := client.VariableSets.Update(ctx, badIdentifier, &VariableSetUpdateOptions{
-			Name:        String("UpdatedName"),
-			Description: String("Updated Description"),
-			Global:      Bool(true),
-		})
-		assert.Nil(t, vsAfter)
-		assert.EqualError(t, err, ErrInvalidVariableSetID.Error())
-	})
-}
-
-func TestVariableSetsUpdateWithPriority(t *testing.T) {
-	skipUnlessBeta(t)
-	client := testClient(t)
-	ctx := context.Background()
-
 	orgTest, orgTestCleanup := createOrganization(t, client)
 	t.Cleanup(orgTestCleanup)
-
 	vsTest, _ := createVariableSet(t, client, orgTest, VariableSetCreateOptions{
 		Name:        String("OriginalName"),
```

Comment:
👋 Hi @Netra2104 - should the string `OriginalName` have a space similar to `line 243` or should `line 243` be updated to read like `line 242`?

Reply:
I think it is fine the way that it is. I have set this test up the same way as other tests. I think descriptions can have spaces hence why it may have been setup this way.